.github/workflows/orchestrator.nms.magmalte.yml

```
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    paths:
+    - 'orchestrator-bundle/nms-magmalte-operator/**'
```

Shouldn't the `-` be intended?
```suggestion
    paths:
      - 'orchestrator-bundle/nms-magmalte-operator/**'
```

---

Actually, I think this applies to all the files. Idk know if this thing would throw a YAML error, but might be worth checking

---

You're right, over half of the yamls had that wrong... yet worked somehow... magic :D
Anyway, according to the docs, it should be indented.
Fixed.